docs: add skip_null_values in configuration, operations and upgrade-g… (#1618)
api-platform/docs: core/upgrade-guide.md

This will be the default value in 3.0, in 2.7 it's left to `true` so that nothing breaks by updating.
By doing so you won't get access to legacy services and this will probably break things on code using `api-platform/core:2.6`.

> In 3.0 the default value of the `skip_null_values` property will be `true`, which means that from now on null values will be omitted when serializing.

Reviewer: ```suggestion
In 3.0 the default value of the `skip_null_values` property is `true`, which means that from now on null values are omitted when serializing.
```

Reviewer 2: maybe can we add somewhere that it's to be in conformance with the JSON Merge Patch RFC and that changing this setting could make the API non-compliant?
